htdocs/core/boxes/box_produits_alerte_stock.php

```diff
@@ -203,7 +203,7 @@ public function loadBox($max = 5)
 
 					$this->info_box_contents[$line][] = array(
 					    'td' => 'class="center"',
-                        'text' => $objp->total_stock.' / '.$objp->seuil_stock_alerte,
+                        'text' => price2num($objp->total_stock, 'MF').' / '.$objp->seuil_stock_alerte,
```

Comment:
'MF' ?
Do you mean 'MS' ?

Reply:
Yes, I made a mistake, it's corrected now